lib/galaxy/jobs/runners/util/cli/job/lsf.py

```diff
@@ -125,5 +129,29 @@ def _get_job_state(self, state):
         except KeyError:
             raise KeyError("Failed to map LSF status code [%s] to job state." % state)
 
+    def _get_excluded_hosts(self):
+        """
+        Reads a file in the set path with one node name per line. All these nodes will be added
+        to the exclusion list for execution.
+
+        The path can be added to destinations like this:
+
+        <destination id="lsf_8cpu_16GbRam" runner="cli">
+            <param id="shell_plugin">LocalShell</param>
+            <param id="job_plugin">LSF</param>
+            <param id="job_memory">16000</param>
+            <param id="job_cores">7</param>
+            <param id="job_excluded_hosts">/path/to/file/with/hosts/to/exclude/one/per/line.txt</param>
+        </destination>
+
+        :param pathExcludedNodes:
+        :return: list with node names
+        """
+        if "excluded_hosts" in self.params:
+            path_excluded = self.params["excluded_hosts"]
+            if path.isfile(path_excluded):
+                return open(path_excluded).read().splitlines()
```

Comment:
```suggestion
                with(open(path_excluded, 'r')) as f:
                    host_names = f.read.splitlines()
                return host_names
```

Reply:
You can even return from within the `with` statement:
```suggestion
                with open(path_excluded) as f:
                    return f.read().splitlines()
```